data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,206 @@
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+mission "Passing Observations"
+	invisible
+	landing
+	destination "Vigales"
+	to offer
+		has "event: introduced scin"
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	on complete
+		conversation
+			`While navigating the skies of this desert world, you notice familiar readings to the desert world you previously encountered a member of the Gegno Vi on. It's likely this world is also home to the same or similar species of creature you confronted back then. Eventually, you come across a large Gegno Vi vessel stationed on the surface, though unlike in your prior experience, the ship isn't sunken into the desert, likely having recently landed here. A group of Vi warriors are marching out and nearby the vessel, and while they do, your radar picks up two more ships descending from above.`
+			choice
+				`	(Leave.)`
+					goto leave
+				`	(Continue to observe.)`
+			`	The approaching ships continue their descent while paying you no attention. They appear to be ones you spotted in orbit when you first entered the system. One of them is the same type of model as the one on the ground, likely some type of transport. The other, however, isn't one you've seen before. It looks like some sort of interceptor vessel, though it is definitely larger than something like a Sparrow from human space, maybe even the size of a Headhunter. Perhaps that ship belongs to someone of importance.`
+			`	In any case, getting mixed up with what may be some sort of military exercise with the Vi could be dangerous, considering that the planet is home to more of those monsters from before. It's best you take your leave now before the situation gets out of control again.`
+
+			label leave
+			`	You decide to withdraw from the area, and the approaching ships continue their way towards the one on the ground as if you weren't present. Regardless of whether or not they were hostile, getting mixed up with what may be some sort of military exercise could be dangerous, especially with the Vi.`
+
+	npc save
+		on kill
+			set "yvx killed"
+		government "Gegno Vi"
+		personality heroic vindictive staying ramming daring uninterested swarming
+		system "Vorsuke"
+		ship "Rhyolite" "Imtydei"
+	npc save
+		government "Gegno Vi"
+		personality heroic vindictive staying uninterested swarming
+		system "Vorsuke"
+		ship "Schist" "Ess Ouec"
+
+
+mission "Foreign Spectator I"
+	invisible
+	landing
+	to offer
+		has "Passing Observations: offered"
+		not "Gegno Genocide Defense: offered"
+	on enter "Eogho"
+		fail
+		event "watcher" 7 14
+	npc
+		on kill
```

Comment:
Should it be `on provoke`?
This is just because in the Vi duel mission, the stuff happens when you provoke them, not when you kill them.
https://github.com/endless-sky/endless-sky/blob/3d017127cecc96c2f912c2d9b011b5d034dc0f08/data/gegno/gegno%20intro%20missions.txt#L563-L588
```suggestion
		on provoke
```
I think `kill` would also mean they can kill each other to fulfill the requirements.